gym_roboy/example_control.py

```diff
@@ -1,34 +1,30 @@
-import gym
-import time
+import os
+
 from stable_baselines.common.policies import MlpPolicy
 
 from stable_baselines.common.vec_env import DummyVecEnv
 from stable_baselines import PPO1
 
-from .envs import MsjEnv, MsjROSBridgeProxy
+from .envs import MsjEnv
 
 
-def our_env_constructor() -> gym.Env:
-    return MsjEnv(MsjROSBridgeProxy())
+MOUNT_DIR = "/root/develDeepAndReinforced/tensorboard_out/"
+MODEL_FILE = os.path.join(MOUNT_DIR, "model")
 
 
 def main():
     # The algorithms require a vectorized environment to run
-    env = DummyVecEnv([our_env_constructor])
-
-    agent = PPO1(MlpPolicy, env, verbose=1)
+    env_constructor = MsjEnv
+    env = DummyVecEnv([env_constructor])
 
-    agent.learn(total_timesteps=1000)
+    if os.path.isfile(MODEL_FILE):
+        agent = PPO1.load(MODEL_FILE)
```

Comment:
Can the loaded agent still learn? I think stable-baseline assumes that loaded agent only be used for prediction.

Reply:
I think it should, but if you want, I can write a test for it. Shall I?

Reply:
I tested. just add the line `agent.set_env(env)`

Reply:
is that really required? The loaded agent does learn, which means that it knows its environment.

Reply:
I'll change the loading logic as discussed